src/StaticValidator.php

```diff
@@ -52,6 +52,12 @@ public static function getPluginManager()
      */
```

Comment:
As per discussion on IRC

* $args param need to improve his description
* $args argument should be renamed